### DISCUSSION TITLE
style max-witdth in table q-select doesn't look good

### DISCUSSION THREAD

**Discussion by CynthiaBlue** [quasarframework/quasar#13663] [CLI - Electron mode]
I have a table with some q-inputs and q-selects in it.  The q-selects have various options, and I want them always to have a max-width.  Without the max-width, they push the table all out of whack.

This is the table with nothing is selected in far-right q-select:
<img width="1055" alt="Screen Shot 2022-06-09 at 2 34 07 PM" src="https://user-images.githubusercontent.com/34698014/172939874-ee9687a9-402c-4d10-a2de-9c05b2ab1aa4.png">

This is what it looks like when something is selected:
<img width="1017" alt="Screen Shot 2022-06-09 at 2 39 50 PM" src="https://user-images.githubusercontent.com/34698014/172940277-2d242a4a-5068-4fae-a9b2-d3fd17113c37.png">

As you can see, the width is being respected, but the text shows outside of the select.  Is there any way to get it truncated, or make it look better?  If I make the select wider, it just pushes everything else so it looks weird.  

My code includes:
```
<q-td key="faults" :props="props">
                      <q-select
                          outlined
                          clearable
                          dense
                          style="max-width: 100px"
                          v-model="props.row.faults"
                          :options="faultOptions"
                          bg-color="teal-2"
                          /> 
                    </q-td>
```

and in the table columns: 
```
{
          name: "faults",
          label: "Faults",
          align: "left",
          headerStyle: 'max-width: 50px',
          field: (row) => row.faults
        },
```

Thanks for feedback.

**Comment by suleimanas** [comment]
in q-select try class="ellipsis"
or wrap the q-select in a dev so it's q-td>div>q-select
or even better remove the clearable btn it takes a lot of the width and use css:
```
.q-select .q-field__native {
  overflow: hidden;
}
```
-or-
```
.q-select .q-field__native span {
  text-overflow: ellipsis;
  overflow: hidden;
}
```
